Cargo.lock

```diff
@@ -114,6 +122,14 @@ dependencies = [
  "proc-macro2 0.4.30 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "quote"
+version = "1.0.0"
```

Comment:
Hmm, so we have two versions of quote and proc-macro2? That seems unfortunate.

Reply:
https://github.com/serde-rs/serde/pull/1591 Unfortunately, serde isn't yet up to date :/

Reply:
There, dupe removed :tada: